src/components/Card.tsx

```diff
@@ -1,43 +1,54 @@
+import { Project } from "@/mock";
 import Image from "next/image";
+import { useRouter } from "next/router";
 import check from "../../public/check.png";
 import sampleIcon from "../../public/favicon/favicon-32x32.png";
 
-
 // TODO
 // dynamic routing on card click
+interface CardProps {
+  project: Project;
+}
 
-const Card = () => {
+const Card = ({ project }: CardProps) => {
+  const router = useRouter();
+  const onCardClick = () => {
+    router.push({
+      pathname: "/project/[id]",
```

Comment:
@berrycakes why is it `project`? I thought it would be a `directory` as indicated in the design doc? Just checking I might miss something.

Reply:
@berrycakes click asks for a review if you are done. Then click resolve conversation if you are confident with your updates.